### PR TITLE
Create supra bars shortcodes and reintroduce navbar example

### PR DESCRIPTION
**Pull request by julien-deramond**
## Previews
* [Design Guidelines](https://deploy-preview-966--boosted.netlify.app/docs/5.1/guidelines/global-headers/)
* [Navbar examples](https://deploy-preview-966--boosted.netlify.app/docs/5.1/examples/navbars/)
* [Docs Orange Navbar](https://deploy-preview-966--boosted.netlify.app/docs/5.1/components/orange-navbar/)

## Remaining tasks
- [x] Modify the 2 example thumbnails
- [x] Fix pa11y issues: https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/actions/runs/1553020422

### CODE REVIEW

File: site/layouts/shortcodes/orange-supra.html [hunk 1]
```diff
@@ -10,6 +14,44 @@
       <li class="nav-item"><a href="#" class="nav-link active" aria-current="page">Personal</a></li>
       <li class="nav-item"><a href="#" class="nav-link">Business</a></li>
     </ul>
-    {{- $input -}}
+{{- if eq $mode "languages" }}
+    <ul class="navbar-nav ml-auto">
```
Comment (louismaximepiton) [inline site/layouts/shortcodes/orange-supra.html:18]: I wonder why there is .ml-auto as this class doesn't exist anymore. The new one is .ms-auto + it works well without it.

File: site/layouts/shortcodes/orange-supra.html [hunk 2]
```diff
@@ -10,6 +14,44 @@
...
+      <li class="nav-item"><a class="nav-link" href="#" aria-label="Version française" lang="fr" hreflang="fr">FR</a></li>
+      <li class="nav-item"><a class="nav-link" href="#" aria-label="Version en español" lang="es" hreflang="es">ES</a></li>
+    </ul>
+{{- end }}
+{{- if eq $mode "actions" }}
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a href="#" class="nav-link nav-icon">
+          <svg fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+            <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#search"/>
```
Comment (louismaximepiton) [inline site/layouts/shortcodes/orange-supra.html:29]: I'd use {{< param docs_version >}} instead of 5.1

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Not sure about two things in the shortcode

File: site/layouts/shortcodes/orange-supra.html (hunk 2 above)
Comment (julien-deramond) [reply site/layouts/shortcodes/orange-supra.html:29]: Yep. I change it for the badge component as well.

File: site/layouts/shortcodes/orange-supra.html (hunk 1 above)
Comment (julien-deramond) [reply site/layouts/shortcodes/orange-supra.html:18]: You're totally right, I remove the useless `.ml-auto` class for supra bars.

**Review by louismaximepiton — APPROVED** [review]

**Pull request merged**